src/sentry/tagstore/migrations/0001_initial.py

```diff
@@ -0,0 +1,235 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't know how or why, but migrations for nodestore at in `south_migrations` folder. Can we make that happen here too?